packages/integration-sdk-runtime/src/logger/index.ts

```diff
@@ -169,7 +169,7 @@ export class IntegrationLogger extends EventEmitter
   implements IntegrationLoggerType {
   private _logger: Logger;
   private _errorSet: Set<Error>;
-  private onFailure: OnFailureFunction;
+  readonly onFailure: OnFailureFunction;
```

Comment:
Made `readonly` so we can call this directly from `IntegrationLogger`. Calling this should not be encouraged though.